src/main/java/com/oltpbenchmark/benchmarks/auctionmark/AuctionMarkLoader.java

```diff
@@ -46,7 +46,7 @@
  * @author pavlo
  * @author visawee
  */
-public class AuctionMarkLoader extends Loader<AuctionMarkBenchmark> {
+public final class AuctionMarkLoader extends Loader<AuctionMarkBenchmark> {
```

Comment:
Many of these classes were converted to `final` to  avoid `this-escape` warnings in which non-final methods are used by the class to initialize the class.  In theory those could be overridden by a subclass to result in problematic initialization.

Since most of these classes are never expected to be subclassed, we simply avoid the error for now and mark them as `final`.

Reply:
Added notes to this effect in the CONTRIBUTING.md file.